ksqldb-serde/src/test/java/io/confluent/ksql/serde/connect/ConnectFormatTest.java

```diff
@@ -95,6 +105,26 @@ public void shouldConvertTransformedConnectSchemaToColumns() {
     assertThat(result.get(1).type(), is(SqlTypes.BIGINT));
   }
 
+  @Test
+  public void shouldSupportPrimitiveSchemas() {
+    // When:
+    format.toParsedSchema(
+        ImmutableList.of(column("bob", SqlTypes.INTEGER)),
+        SerdeOptions.of(SerdeOption.UNWRAP_SINGLE_VALUES),
+        formatInfo
+    );
+
+    // Then:
+    assertThat(capturedConnectSchema, is(SchemaBuilder.int32().optional().build()));
+  }
+
+  private static SimpleColumn column(final String name, final SqlType sqlType) {
+    final SimpleColumn column = mock(SimpleColumn.class);
+    when(column.name()).thenReturn(ColumnName.of(name));
+    when(column.type()).thenReturn(sqlType);
+    return column;
+  }
+
```

Comment:
Add two more tests to validate the negative cases?
```
if (unwrapSingle && !singleField) {
      throw new KsqlException("'" + CommonCreateConfigs.WRAP_SINGLE_VALUE + "' "
          + "is only valid for single-field value schemas");
    }
```

Reply:
That code added to `ConnectFormat` will be removed again in the next PR, (its code duplication at the moment), hence no tests.